src/styles/blocks/function.scss

```diff
@@ -0,0 +1,17 @@
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+  }
+}
+
+@each $day, $shift in $week {
```

Comment:
The variable `$week` is used in the `@each` loop but is not defined in this file. Ensure it is defined or imported from another SCSS file.